README.md

```diff
@@ -178,405 +128,133 @@ After installing the library you will need to require it in your test file (see
-        browser.get("https://example.com");
-
-        const textField = element(by.css("input.some-text-field"));
-
-        protractorHelper.fillFieldWithTextWhenVisible(textField, "some text", "textField is not visible", 3000);
-
-        // ...
-    });
-});
-```
-
-### Example of usage of `fillInputFieldWithFileWhenPresent`
-
-```js
-const path = require("path");
-const protractorHelper = require("protractor-helper");
-
-describe("foo", () => {
-    it("bar", () => {
-        const fileToUpload = "../assets/someFile.png";
-        const absolutePathOfFileToUpload = path.resolve(__dirname, fileToUpload);
-
-        browser.get("https://example.com");
-
-        const fileInputField = element(by.css("input.file-input"));
-
-        protractorHelper.fillInputFieldWithFileWhenPresent(fileInputField, absolutePathOfFileToUpload, "fileInputField is not present", 3000);
-
-        // ...
-    });
-});
-```
-
-### Example of usage of `clearFieldWhenVisible`
-
-```js
-const protractorHelper = require("protractor-helper");
-
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
-
-        const textField = element(by.css("input.some-text-field"));
-
-        textField.sendKeys("foobar");
-        protractorHelper.clearFieldWhenVisible(textField, "textField is not visible", 3000);
-
-        // ...
-    });
-});
-```
-
-### Example of usage of `clearFieldWhenVisibleAndFillItWithText`
-
-```js
-const protractorHelper = require("protractor-helper");
-
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
-
-        const textField = element(by.css("input.some-text-field"));
-
-        protractorHelper.clearFieldWhenVisibleAndFillItWithText(textField, "some text", "textField is not visible", 3000);
-
-        // ...
-    });
-});
-```
-
-### Example of usage of `tapWhenTappable`
-
-```js
-// Imagine that in the `protractor.conf.js` file a mobile emulator is being defined.
-
-const protractorHelper = require("protractor-helper");
-
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
-
-        const myButton = element(by.id("my-button"));
-
-        protractorHelper.tapWhenTappable(myButton, "myButton is not tappable", 3000);
+## Available helpers
 
-        // ...
-    });
-});
-```
+Below is the list of all available helpers in this library with your respective example:
 
-### Example of usage of `waitForTextToBePresentInElement`
+### `setTimeout`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method allows to change the timeout duration of all `protractor-helper` methods called below `setTimeout`.
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+The `timeoutInMilliseconds` default is 5000 milliseconds
 
-        const myButton = element(by.id("my-button"));
+If is called without pass argument the timeout of the tests is changed to the default timeout.
```

Comment:
What about 'If called without passing an argument the timeout will be set to the default one'.

Reply:
Commit made because this review: f76a828